src/SectionTitle.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import cx from 'classnames';
+import Chunk from './Chunk';
+import Flex from './Flex';
+import FlexItem from './FlexItem';
+
+export const SECTIONTITLE_CLASS = 'sectionTitle';
+/**
+ * @module SectionTitle
+ */
+class SectionTitle extends React.Component {
+	render() {
+		const {
+			action,
+			className,
+			title,
+			...other
+		} = this.props;
+
+		const classNames = cx(
+			SECTIONTITLE_CLASS,
+			className
+		);
+
+		return (
+			<Flex
+				align='center'
+				className={classNames}
+				{...other}
+				>
+				<FlexItem>
```

Comment:
I was talking with Mike P about this a couple weeks ago, asking about cases where we want chunk-like functionality on a non-chunk component, and he suggested that in those cases you might just want a `margin--bottom` className on the outer component, in this case the `<Flex>`. I think that expresses the layout intention reasonably clearly without having to worry about awkward/redundant layout composition. Just an option.